routes/prize_route.go

```diff
@@ -0,0 +1,20 @@
+package routes
+
+import (
+	"github.com/drift-org/backend/controllers"
+	"github.com/gin-gonic/gin"
+)
+
+var (
+	prizeController controllers.PrizeController = controllers.NewPrizeController()
+)
+
+/*
+ For now, authentication is not necessary to create a prize.
```

Comment:
I think we should still use auth middleware, since an admin will still pass auth. In the future, what we can do is create another middleware that checks that the authenticated user is an admin. But i think its unsafe to leave out the auth middleware here.

Reply:
Sure, only downside is we'll have to unnecessarily create dummy user & login just for Prize testing even when it has no functionality right now.

Reply:
Gonna hold off on this until Aiden's PR is merged (since there are some changes for testing helpers, and I can't cherrypick one commit w/o pulling some unwanted changes).

Reply:
> Sure, only downside is we'll have to unnecessarily create dummy user & login just for Prize testing even when it has no functionality right now.


Not sure why you have to do this? You can have it in the route, but you don't use it in the controller at all?

Reply:
Yep unit tests. Never mind then, forgot that tests are only looking at the controllers (though this means we might want to figure out how we can test routes as well, for the ones that are auth-protected).

Will just address this right now then & not change the test.

Reply:
> Yep unit tests. Never mind then, forgot that tests are only looking at the controllers (though this means we might want to figure out how we can test routes as well, for the ones that are auth-protected).

We decided that we are testing middle ware separately (yet to be wrritten, i'll make issues). Other than that, we don't need to test every route that is auth-protected, just the ones that use the userID from the middleware. We also decided that routes will not be tested.